packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -49,9 +64,35 @@ class SelectableTableRows extends React.Component {
     const initialOnClick = row.props.onClick;
 
     return (event) => {
-      // The default isSelectable attribute is either undefined or true, unless the consumer specifies the row's isSelectable attribute as false.
       if (row.props.isSelectable !== false) {
         this.handleOnChange(event, index);
+
+        const { liveRegion } = this.props;
+        const { intl } = this.context;
+
+        if (this.props.screenReaderUpdateOnSelected) {
+          this.props.screenReaderUpdateOnSelected(liveRegion, intl);
+        } else {
+          const isSelected = this.props.selectedIndexes.indexOf(index) >= 0;
+
+          // The isSelected value here actually refers to the previous state of the row since
```

Comment:
This comment doesn't seem to belong here.

Reply:
https://github.com/cerner/terra-core/pull/2331/commits/3fb9b80ad6657aec97c4c26e072c7f421c0e62fc